cluster_cmd

```diff
@@ -0,0 +1,15 @@
+# The following commands can be issued at a python prompt to setup a
```

Comment:
One could also run this as a python script, so maybe add `.py` extension?

Reply:
Running this as a python script would set up a cluster with a scheduler, but the client would be lost as soon as the script exits. The Script would nit, however, provide any information on where to reach the scheduler.

If one wants to accomplish this, setting up a cluster from the command line with a script, the better option would be to use the disk-ssh command line script, which is installed along with dask.

The purpose of this, was to have quick reference for the relevant commands from within a pythin script or module.